src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div>
+    <div>Display</div>
+    <ButtonPanel />
+  </div>
```

Comment:
* [ ] it should be placed in `src/components` directory ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/App.png))
* [ ] it should render the `Display` component ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/appFunctionalComponent.png))
* [ ] it should have children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))